src/CarbonAware.DataSources/CarbonAware.DataSources.Json/src/JsonDataSource.cs

```diff
@@ -23,18 +24,24 @@ public class JsonDataSource : ICarbonIntensityDataSource
 
     private List<EmissionsData>? emissionsData;
 
-    private readonly ILogger<JsonDataSource> _logger;
```

Comment:
This is a private variable, so I suppose the variable name should be camel case with _ ? Just the way it was before?

Reply:
Thanks! yeah, I think going forward we should follow the official conventions as much as possible - https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions